.github/workflows/docker_firedrake-um2n.yml

```diff
@@ -0,0 +1,39 @@
+name: 'Build bespoke UM2N Docker container'
+
+on:
+  # Check that Docker build succeeds when these files get changed in an open PR
+  pull_request:
+    paths:
+      - '.github/workflows/docker_firedrake-um2n.yml'
+      - 'docker/Dockerfile.firedrake-um2n'
+
+  # Build and push the Docker container when these files get changed on the main branch
+  push:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/docker_firedrake-um2n.yml'
+      - 'docker/Dockerfile.firedrake-um2n'
+
+  # Trigger this workflow when docker_firedrake-um2n.yml completes on the main branch
+  workflow_run:
+    workflows: ['Build bespoke UM2N Docker container']
+    types:
+      - completed
+    branches:
+      - main
```

Comment:
Saw this now so just commenting immediately :) `workflow_run` with `completed` type triggers the workflow to run when another workflow has finished. So in the Firedrake workflow I put this because the Firedrake image depends on the PETSc base image. But the UM2N image doesn't depend on any so we can remove this. Maybe my description in the comment isn't very clear

Reply:
Fixed in https://github.com/mesh-adaptation/mesh-adaptation-docs/pull/59/commits/548fe29ccc593c4094de299e2522fd6ecff8fd5e.